Assets/StreamingAssets/Images/Furniture/Vent.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Sprites>
+	<Sprite name="vent"    x="0" y="0" w="1" h="1" pixelPerUnit="64" />
```

Comment:
Bloody Notepad++ no matter how much I tell it not to, it keeps sneaking tabs in when I'm not looking.